depthai_bridge/urdf/include/depthai_macro.urdf.xacro

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <robot name="depthai_camera" xmlns:xacro="http://ros.org/wiki/xacro">
 
-<xacro:macro name="depthai_camera" params="camera_name camera_model parent base_frame cam_pos_x cam_pos_y cam_pos_z cam_roll cam_pitch cam_yaw">
+<xacro:macro name="depthai_camera" params="tf_prefix camera_model parent base_frame cam_pos_x cam_pos_y cam_pos_z cam_roll cam_pitch cam_yaw">
```

Comment:
lets keep it as camera_name in the urdf since it will be the name of the camera when multiple cameras are used. And use TF-Prefix when writing the code. that way. when someone integrates our device into their model. they won't have same naming issues